server/commands.py

```diff
@@ -4940,7 +4834,9 @@ def ooc_cmd_switch(client: ClientManager.Client, arg: str):
     /switch Phoenix_HD
     """
 
-    if len(arg) == 0:
+    try:
+        Constants.assert_command(client, arg, parameters='>1')
```

Comment:
This fails on characters that have no spaces (e.g. /switch Phoenix_HD, /switch SPECTATOR)